package.json

```diff
@@ -36,11 +36,10 @@
     "url": "https://github.com/IBM/node-sdk-core/issues"
   },
   "devDependencies": {
-    "@microsoft/api-documenter": "^7.13.68",
-    "@microsoft/api-extractor": "^7.18.19",
+    "@microsoft/api-documenter": "^7.13.76",
+    "@microsoft/api-extractor": "^7.19.2",
     "@types/debug": "^4.1.7",
     "@types/lodash.isempty": "^4.4.6",
-    "@types/dotenv": "^8.2.0",
```

Comment:
Messages displayed by npm when running "npm install" indicate that this dependency is simply not needed.